docs/manual/docs/cmd/graph/teams/teams-user-add.md

```diff
@@ -0,0 +1,42 @@
+# graph teams user add
+
+Adds an owner or member to the specified Microsoft Teams team
+
+## Usage
+
+```sh
+graph teams user add [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-i, --teamId <teamId>`|The ID of the team for which to list users
+`-n, --userName <userName>`|User\'s UPN (user principal name - e.g. johndoe@example.com)
+`-r, --role <type>`|Filter the results to only users with the given role: `Owner|Member`
```

Comment:
Incorrect description. Should be _The role to be assigned to the new user: `Owner|Member`. Default `Member`_